manifests-maistra/2.0.0.2/maistraoperator.v2.0.0.2.clusterserviceversion.yaml

```diff
@@ -12,7 +12,7 @@ metadata:
       The Maistra Operator enables you to install, configure, and manage an instance of Maistra service mesh. Maistra is based on the open source Istio project.
 
     containerImage: maistra/istio-ubi8-operator:2.0.0
-    createdAt: 2020-11-10T03:22:53UTC
+    createdAt: 2020-12-01T15:05:30EST
```

Comment:
Can we remove the csv files from the commit? They have a tendency to cause merge issues when the files actually change.

Reply:
@rcernich I'm assuming those timestamps are ignored by the check-gen job?

Reply:
`make-gen` should be smart enough to not make this change is this is the case